NuCivic/dkan
# Added status field to harvest event table #CIVIC-4504

Issue: https://jira.govdelivery.com/browse/CIVIC-4504
## Description

We need a new column on the table for event table on harvest pages. 
## How to reproduce
1.  Go and create a Harvest Source using a JSON that will produce a simple error when it gets imported like having a "00:00" value on the Boureau Code field.
   Harvest the source.
2. Go to the harvest dashboard and confirm that you see the "Error" mark on the source row.
3. Harvest the source again with no changes on the JSON file.
4. Go to the harvest dashboard and confirm that you see the "OK" mark on the source row.
5. Go to the source page and click on "Events"
6. Notice that you have 2 events listed there but you cannot tell which one of those had errors and which one didn't.
## QA Tests
- [ ] Create a harvest source and import content (With bad data and with fine data)
- [ ]  'Status' column exists to the right of the 'Date' column (event page)
- [ ] A red circle is displayed in the 'Status' column for each event that had an error, mouse over tool tip 'Error', alt text 'Error'
- [ ] Click on a red circle to navigate to the error log
- [ ] A green circle is displayed in the 'Status' column for each event that had no errors, mouse over tool tip 'OK', alt text 'OK'
